pallets/did/src/origin.rs

```diff
@@ -123,13 +123,12 @@ where
 	}
 }
 
-#[cfg(all(test, feature = "runtime-benchmarks"))]
+#[cfg(test)]
 mod tests {
-	use super::EnsureDidOrigin;
-
+	#[cfg(feature = "runtime-benchmarks")]
 	#[test]
```

Comment:
This seems a bit odd to me

Reply:
Was it previously, throwing warnings?

Reply:
Yes, because it meant the `tests` module was not used only for tests but also for something else, and it was indeed throwing a lint. Ideally I agree with the lint, a `tests` module should only be used for tests. Then if something from inside the module is used when testing some feature, then it should be feature-gated inside the module.

Reply:
I see